src/dataDisplay/Icon/index.tsx

```diff
@@ -149,19 +161,39 @@ const icons = {
 };
 
 export type IconType = typeof icons;
+export type IconTypes = keyof IconType;
 
 type Props = {
-  type: keyof IconType;
-  color?: keyof Theme['colors'];
+  type: IconTypes;
   size: keyof Theme['icons']['size'];
+  color?: keyof Theme['colors'];
```

Comment:
I see we have too many places where we write this:
```
keyof Theme['icons']['size']
```

we could just write somewhere
```
export type Sizes = keyof Theme['icons']['size'];
```
and import it. Same applies to colors.

Reply:
good idea